drivers/interrupt_controller/intc_gicv3.c

```diff
@@ -249,14 +253,18 @@ static void gicv3_dist_init(void)
 		sys_write32(BIT_MASK(GIC_NUM_INTR_PER_REG),
 			    ICPENDR(base, idx));
 		/* All SPIs are G1S and owned by Zephyr */
-		sys_write32(0, IGROUPR(base, idx));
+		sys_write32(~0, IGROUPR(base, idx));
```

Comment:
The comment needs to be updated

Reply:
What happens if Zephyr is running in NS mode?

Reply:
The TODOs in this driver roughly marks where we need to update to support NS state. 
Apart from the GICD we need CPU interface routed accordingly and conditionally. We dont want to drop support for 
running Zephyr in Secure state as a standalone RTOS ( start EL3 ->run EL1S).
ref: https://github.com/zephyrproject-rtos/zephyr/blob/0277b4f071818e836ac4ed1631abfa425a9dd7e9/drivers/interrupt_controller/intc_gicv3.c#L183-L189

Reply:
ok. Then need to introduce an extra Kconfig option for zephyr in secure and non-secure stage.